src/main/java/io/github/bookcrawler/core/impl/publio/PublioBookInfoParser.java

```diff
@@ -0,0 +1,53 @@
+package io.github.bookcrawler.core.impl.publio;
+
+import io.github.bookcrawler.core.BookInfoParser;
+import io.github.bookcrawler.core.impl.SourceScrappingResult;
+import io.github.bookcrawler.entities.BookInfo;
+import io.github.bookcrawler.entities.BookInfoBuilder;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+
+import static io.github.bookcrawler.core.BookStore.PUBLIO;
+
+public class PublioBookInfoParser implements BookInfoParser {
+
+    @Override
+    public BookInfo parse(SourceScrappingResult sourceScrappingResult) {
+        Document source = sourceScrappingResult.getSource();
+        return new BookInfoBuilder()
+                .title(parseTitle(source))
+                .author(parseAuthor(source))
+                .description(parseDescription(source))
+                .price(parsePrice(source))
+                .library(PUBLIO.toString())
+                .url(source.location())
+                .build();
```

Comment:
In the end, wasn't it supposed to be builder moved to ACL so JSoup abstractions were contained?